gallery/visualtests/nxModal.js

```diff
@@ -22,12 +22,13 @@ describe('NxModal', function() {
       narrowExampleSelector = '#nx-modal-narrow-example',
       wideExampleSelector = '#nx-modal-wide-example',
       stackedExampleSelector = '#nx-modal-stacked-example',
+      nestedExampleSelector = '#nx-modal-nested-example',
       escClosingExampleSelector = '#nx-modal-esc-example';
 
   async function openModal(exampleSelector) {
     const openModalBtnSelector = `${exampleSelector} button`,
         closeModalBtnSelector =
-            `${exampleSelector} .nx-footer .nx-btn-bar .nx-btn:not(.nx-btn--primary):not(.nx-btn-tertiary)`;
+        `${exampleSelector} .nx-footer .nx-btn-bar .nx-btn:not(.nx-btn--primary):not(.nx-btn-tertiary)`;
```

Comment:
Remaining autoformat changes

Reply:
> Remaining autoformat changes

Thanks fixed: bc426e2939b3d22c6d9e91dcf0b72563b966cccc

Reply:
> Thanks fixed: https://github.com/sonatype/sonatype-react-shared-components/commit/bc426e2939b3d22c6d9e91dcf0b72563b966cccc

There was more than just that line.  Check the overall PR diff

Reply:
> > Thanks fixed: [bc426e2](https://github.com/sonatype/sonatype-react-shared-components/commit/bc426e2939b3d22c6d9e91dcf0b72563b966cccc)
> 
> There was more than just that line. Check the overall PR diff

Ops, fixed: c23e1604fde46358a504a4d6a25a9bdafefb0c41